include/z64.h

```diff
@@ -1689,6 +1689,13 @@ typedef struct DebugDispObject {
     /* 0x2C */ s32 pad; //Padding not in the OOT version
 } DebugDispObject; // size = 0x30
 
+typedef struct {
+    Actor* unk_00;
+    f32 unk_04;
+} z_sub_unkStruct_8013E640;
+
+typedef s32 (*func_8013E640_arg6)(s32, Actor*, Actor*, z_sub_unkStruct_8013E640*);
```

Comment:
```suggestion
typedef s32 (*func_8013E640_arg6)(GlobalContext*, Actor*, Actor*, void*);
```
Should just be taken care of when you merge in master.